addons/barcodes/static/src/barcode_handler_field.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { useBus, useService } from "@web/core/utils/hooks";
+
+const { Component, xml } = owl;
+
+export class BarcodeHandlerField extends Component {
+    setup() {
+        this.barcode = useService("barcode");
```

Comment:
really small detail, but if you don't need to keep a reference to the barcode, you can just assign it to a local variable:
```js
const barcode = useService("barcode");
useBus(barcode.bus, ...);
```

Reply:
I agree to that.